packages/orbit-components/src/primitives/TooltipPrimitive/TooltipPrimitive.ct.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import { test, expect } from "@playwright/experimental-ct-react";
+
+import { TooltipPrimitiveVisualDefaultStory } from "./TooltipPrimitive.ct-story";
+
+export const resizeViewportHeight = async page => {
+  const viewportSize = page.viewportSize();
+
+  if (viewportSize) {
```

Comment:
This function sets a larger height of the viewport, without that, I couldn't cover all of the tooltip placements in one screen, eg: in the picture below, the last row of the tooltips are indented, because they were out of the default viewport height:
![image](https://github.com/user-attachments/assets/5cb30279-dd63-404d-b0f8-0bc4e1f1f0fd)